posix-mac-addons/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+cmake_minimum_required(VERSION 3.13)
```

Comment:
I will not comment on any of the `posix-mac-addons` now. The cleanups and further improvements are tracked here: https://github.com/stanislaw/posix-mac-addons/issues. For the time being, I will be force-pushing or adding some cleanups to this branch while also re-running the tests with the test `Makefile` from this changeset.